symbology/read-symbols-from-mobile-style-file/src/main/java/com/esri/samples/read_symbols_from_mobile_style_file/ReadSymbolsFromMobileStyleFileController.java

```diff
@@ -116,20 +114,12 @@ private void loadSymbolsFromStyleFile() {
       }
 
       // load the default search parameters
-      ListenableFuture<SymbolStyleSearchParameters> defaultSearchParametersFuture = emojiStyle.getDefaultSearchParametersAsync();
-      defaultSearchParametersFuture.addDoneListener(() -> {
-        try {
-          SymbolStyleSearchParameters defaultSearchParameters = defaultSearchParametersFuture.get();
-
-          // use the default parameters to perform the search, getting all the available symbols within the file
-          ListenableFuture<List<SymbolStyleSearchResult>> symbolStyleSearchResultFuture = emojiStyle.searchSymbolsAsync(defaultSearchParameters);
-          symbolStyleSearchResultFuture.addDoneListener(() -> {
-            try {
-
-              // loop through the results and add each item to a list view according to category
-              List<SymbolStyleSearchResult> symbolStyleSearchResults = symbolStyleSearchResultFuture.get();
-              symbolStyleSearchResults.forEach(symbolStyleSearchResult -> {
-
+      emojiStyle.getDefaultSearchParametersAsync().toCompletableFuture()
```

Comment:
```suggestion
     // insert doc here
      emojiStyle.getDefaultSearchParametersAsync().toCompletableFuture()
```
I think we need some more explicit doc here